aws-kafkaconnect-connector/docs/README.md

```diff
@@ -0,0 +1,195 @@
+# AWS::KafkaConnect::Connector
+
+Resource Type definition for AWS::KafkaConnect::Connector
+
+## Syntax
+
+To declare this entity in your AWS CloudFormation template, use the following syntax:
+
+### JSON
+
+<pre>
+{
+    "Type" : "AWS::KafkaConnect::Connector",
+    "Properties" : {
+        "<a href="#capacity" title="Capacity">Capacity</a>" : <i><a href="capacity.md">Capacity</a></i>,
+        "<a href="#connectorconfiguration" title="ConnectorConfiguration">ConnectorConfiguration</a>" : <i><a href="connectorconfiguration.md">ConnectorConfiguration</a></i>,
+        "<a href="#connectordescription" title="ConnectorDescription">ConnectorDescription</a>" : <i>String</i>,
+        "<a href="#connectorname" title="ConnectorName">ConnectorName</a>" : <i>String</i>,
+        "<a href="#kafkacluster" title="KafkaCluster">KafkaCluster</a>" : <i><a href="kafkacluster.md">KafkaCluster</a></i>,
+        "<a href="#kafkaclusterclientauthentication" title="KafkaClusterClientAuthentication">KafkaClusterClientAuthentication</a>" : <i><a href="kafkaclusterclientauthentication.md">KafkaClusterClientAuthentication</a></i>,
+        "<a href="#kafkaclusterencryptionintransit" title="KafkaClusterEncryptionInTransit">KafkaClusterEncryptionInTransit</a>" : <i><a href="kafkaclusterencryptionintransit.md">KafkaClusterEncryptionInTransit</a></i>,
+        "<a href="#kafkaconnectversion" title="KafkaConnectVersion">KafkaConnectVersion</a>" : <i>String</i>,
+        "<a href="#logdelivery" title="LogDelivery">LogDelivery</a>" : <i><a href="logdelivery.md">LogDelivery</a></i>,
```

Comment:
A thought here:
I'll need to click the link double times to understand syntax of log delivery.
If we look at our [api document](https://docs.aws.amazon.com/MSKC/latest/mskc/API_CreateConnector.html), we list the log delivery full json in the same page. I think it would be better to dump the log delivery json/yaml here, as we only provide one kind of log to cx.

Or maybe we give an example of entire json/yaml with all essential settings.

Do we need to engage the msk tech writer to review these?

Reply:
the documentation is automatically generated via `cfn generate`, and is based on the `aws-kafkaconnect-connector.json` file